ckanext/xloader/utils.py

```diff
@@ -9,7 +9,7 @@
 from decimal import Decimal
 
 import ckan.plugins as p
-from ckan.plugins.toolkit import config
+from ckan.plugins.toolkit import config, h, _
```

Comment:
`_` is used as a loop variable inside `type_guess`, Flake8 doesn't like us having two variables with the same name.